### PR TITLE
Add /suggestions and /bugs

### PR DESCRIPTION
**Pull request by Rhythms88**
- `/suggestions` is for whenever a user in lobby/help/another room says
something like "I think x would be good to add" so someone can just do
`!suggestions` or w/e
- `/bugs` is for when someone says "I found a bug that relates to x" (or
something of the sort) and a voice can do `!bugs` or `!bugreport`

### CODE REVIEW

**Comment by Rhythms88** [comment]
Something else regarding `/suggestions` and `/bugs`:

[19:07] Rhythms: does anyone think a command to link to the suggestions page would be beneficial?
[19:07] Rhythms: EX: Someone says that they have a suggestion for the server in Lobby/Help (or any other room fsr), people could just do something like !suggestions
[19:07] %SolarisFox: How am I supposed to procrastinate figuring out how to make /itemsearch work well if you keep doing all of the easy stuff??
[19:08] Rhythms: and it would give the user a place to post their suggestion
[19:08] Rhythms: etc
[19:08] Rhythms: SolarisFox ruined my thought process,,,,
[19:08] %SolarisFox: lobby roomintro already has suggestion link in it
[19:08] Rhythms: so
[19:08] %SolarisFox: they could just !roomintro and say to click that
[19:09] Rhythms: i dont think someone will be spamming !roomintro every time someone has a suggestion
[19:09] Rhythms: and the lobby roomintro isnt every romintro
[19:09] Rhythms: roomintro*
<hr>
[19:12] &Slayer95: I support /suggestions
[19:12] #Zarel: !_!
[19:12] &Slayer95: also /bugs

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1510,7 +1524,7 @@ var commands = exports.commands = {
 	intro: function (target, room, user) {
 		if (!this.canBroadcast()) return;
 		this.sendReplyBox(
-			"New to competitive pokemon?<br />" +
+			"New to competitive Pokemon?<br />" +
```
Comment (Bryan-0) [inline chat-plugins/info.js]: I think you forget the ``\u00e9`` here, "Pok\u00e9mon" instead of "Pokemon".

Comment (Rhythms88) [inline chat-plugins/info.js]: i didnt mean to put this in the PR so...

Comment (Rhythms88) [inline chat-plugins/info.js]: You can make the PR if you want @bryan-0

File: chat-plugins/info.js [hunk 2]
```diff
@@ -1499,6 +1499,20 @@ var commands = exports.commands = {
...
+	suggestions: function (target, room, user) {
+		if (!this.canBroadcast()) return;
+		this.sendReplyBox("<a href=\"https://www.smogon.com/forums/threads/3534365/\">Make a suggestion for Pok&eacute;mon Showdown</a>");
+	},
+
+	bugreport: 'bugs',
+	bugs: function (target, room, user) {
+		if (!this.canBroadcast()) return;
+		this.sendReplyBox(
+			"Found a bug on Pok&eacute;mon Showdown that you would like to report?<br />" +
```
Comment (Slayer95) [inline chat-plugins/info.js]: Maybe ``Have a replay showcasing a bug on Pok&eacute;mon Showdown?``

**Comment by Slayer95** [comment]
Eeeeh, you don't need to close this :|

**Comment by Slayer95** [comment]
Oh, unknown repository; rip

**Comment by Rhythms88** [comment]
I accidentally deleted my repository that's why :(

**Pull request closed without merging**